core_backend/app/db/db_models.py

```diff
@@ -390,50 +584,143 @@ async def update_content_in_db(
 
 
 async def delete_content_from_db(
```

Comment:
Can we rename these functions to `delete_content_text_from_db` etc? Make explicit we're dealing with content_text, not content?